### PR TITLE
Small Improvements

### PR DESCRIPTION
**Pull request by xandao6**
Changed file names based on Guido's convention
Changed some selectors
Updated the scripts
Kill Chrome after program close

### CODE REVIEW

**Comment by rpotter12** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 3
           

Complexity increasing per file
==============================
- wplay/utils/target_data.py  2
- wplay/utils/browser_config.py  4
- wplay/utils/target_search.py  7
         

Complexity decreasing per file
==============================
+ wplay/onlinetracker.py  -1
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/rpotter12/whatsapp-play/pullRequest?prid=5051926&bid=16632931)

File: wplay/utils/helpers.py [hunk 1]
```diff
@@ -10,13 +10,19 @@
...
-    'search_contact_input': '#app > div > div span > div > span > div div > label > input',
-    'contact_list_elements_filtered': '#app > div > div span > div > span > div div > div div > div div > span > span[title][dir]',
-    'group_list_elements_filtered': '#app > div > div span > div > span > div div > div div > div div > span[title][dir]',
+    'search_contact_input_new_chat': '#app > div > div span > div > span > div div > label > input',
+    'contact_list_elements_filtered_new_chat': '#app > div > div span > div > span > div div > div div > div div > span > span[title][dir]',
+    'group_list_elements_filtered_new_chat': '#app > div > div span > div > span > div div > div div > div div > span[title][dir]',
+    
+    'search_contact_input':'#side > div > div > label > input',
+    'chat_list_elements_filtered':'#pane-side > div > div > div > div > div > div > div > div > div > span > span[title][dir]',
+
```
Comment (rpotter12) [inline wplay/utils/helpers.py]: Codacy Issue found: [Trailing whitespace](https://app.codacy.com/manual/rpotter12/whatsapp-play/pullRequest?prid=5051926)

File: wplay/utils/target_data.py [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+# endregion
+
+
+# region FOR SCRIPTING
+async def get_last_seen_from_focused_target(page):
+    #await page.waitForSelector(whatsapp_selectors_dict['status'], visible = True)
+    try:
+        status = await page.evaluate(f'document.querySelector("{whatsapp_selectors_dict["last_seen"]}").getAttribute("title")')
+        return status
+    except:
```
Comment (rpotter12) [inline wplay/utils/target_data.py]: Codacy Issue found: [No exception type(s) specified](https://app.codacy.com/manual/rpotter12/whatsapp-play/pullRequest?prid=5051926)

File: wplay/utils/target_data.py [hunk 3]
```diff
@@ -0,0 +1,15 @@
+
+# region IMPORTS
+from wplay.utils.helpers import whatsapp_selectors_dict, websites
```
Comment (rpotter12) [inline wplay/utils/target_data.py]: Codacy Issue found: [Unused websites imported from wplay.utils.helpers](https://app.codacy.com/manual/rpotter12/whatsapp-play/pullRequest?prid=5051926)

**Comment by nileshpatra** [comment]
I wish to suggest two things:

* Unused + Commented variables (line 6, 12 in what.py) can be removed
* Since this isn't a major change, it will be a good idea to 'squash' and preserve author information in previous commits

**Comment by nileshpatra** [comment]
@rpotter12 : request to you to squash + merge in every PR, including this

**Comment by rpotter12** [comment]
> @rpotter12 : request to you to squash + merge in every PR, including this

Thanks @nileshpatra. It is a good suggestion. All the contributors should squash their commits.

**Comment by rpotter12** [comment]
@xandao6 please squash your commits. :)

**Comment by danilofuchs** [comment]
> @rpotter12 : request to you to squash + merge in every PR, including this

I agree. Squashing PRs when merging greatly increases repo's maintainability and ability to rebase any feature branch.

> @xandao6 please squash your commits. :)

But, I don't think @xandao6 needs to do the squash himself, as the PR commit history will be lost. I rather the maintainer do a squash+merge via GitHub's UI

File: wplay/utils/helpers.py (hunk 1 above)
Comment (rpotter12) [inline wplay/utils/helpers.py]: Codacy Issue found: [Trailing whitespace](https://app.codacy.com/manual/rpotter12/whatsapp-play/pullRequest?prid=5051926)

**Comment by rpotter12** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 2
           

Complexity increasing per file
==============================
- wplay/utils/browser_config.py  4
- wplay/utils/target_data.py  2
- wplay/utils/target_search.py  7
- wplay/utils/kill_process.py  3
         

Complexity decreasing per file
==============================
+ wplay/onlinetracker.py  -1
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/rpotter12/whatsapp-play/pullRequest?prid=5051926&bid=16632931)

File: wplay/utils/target_data.py [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+# endregion
+
+
+# region FOR SCRIPTING
+async def get_last_seen_from_focused_target(page):
+    #await page.waitForSelector(whatsapp_selectors_dict['status'], visible = True)
+    try:
+        status = await page.evaluate(f'document.querySelector("{whatsapp_selectors_dict["last_seen"]}").getAttribute("title")')
+        return status
+    except:
```
Comment (rpotter12) [inline wplay/utils/target_data.py]: Codacy Issue found: [No exception type(s) specified](https://app.codacy.com/manual/rpotter12/whatsapp-play/pullRequest?prid=5051926)

**Comment by rpotter12** [comment]
> I wish to suggest two things:
> 
> * Unused + Commented variables (line 6, 12 in what.py) can be removed
> * Since this isn't a major change, it will be a good idea to 'squash' and preserve author information in previous commits

@xandao6 Please remove the unused + commented variables

**Comment by rpotter12** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 1
- Added 3
           

Complexity increasing per file
==============================
- wplay/utils/kill_process.py  3
- wplay/utils/browser_config.py  4
- wplay/utils/target_data.py  2
- wplay/utils/target_search.py  7
         

Complexity decreasing per file
==============================
+ wplay/onlinetracker.py  -1
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/rpotter12/whatsapp-play/pullRequest?prid=5051926&bid=16632931)

File: wplay/utils/helpers.py (hunk 1 above)
Comment (rpotter12) [inline wplay/utils/helpers.py]: Codacy Issue found: [Trailing whitespace](https://app.codacy.com/manual/rpotter12/whatsapp-play/pullRequest?prid=5051926)

File: wplay/utils/target_data.py (hunk 4 above)
Comment (rpotter12) [inline wplay/utils/target_data.py]: Codacy Issue found: [No exception type(s) specified](https://app.codacy.com/manual/rpotter12/whatsapp-play/pullRequest?prid=5051926)

File: wplay/messagetimer.py [hunk 5]
```diff
@@ -36,7 +28,7 @@
     for _ in range(number_of_messages):
         if not messages:
             break
-        await pypIO.send_message(page, messages[random.randrange(0, message_type_numbers)])
+        await io.send_message(page, messages[random.randrange(0, message_type_numbers)])
```
Comment (rpotter12) [inline wplay/messagetimer.py]: Codacy Issue found: [Standard pseudo-random generators are not suitable for security/cryptographic purposes.](https://app.codacy.com/manual/rpotter12/whatsapp-play/pullRequest?prid=5051926)

**Comment by nileshpatra** [comment]
@rpotter12 please
* Remove [this](https://github.com/rpotter12/whatsapp-play/commit/2afb8ee5e748232ef6b18f403fbb44397b9ce99c#comments) it isn't helpful + probably one of the reason for circleci build to fail
* Re-trigger the circle ci build for this PR? It has a weird error at the moment

**Comment by rpotter12** [comment]
> * Remove [this](https://github.com/rpotter12/whatsapp-play/commit/2afb8ee5e748232ef6b18f403fbb44397b9ce99c#comments) it isn't helpful + probably one of the reason for circleci build to fail

The above commit had to be made since when https://github.com/rpotter12/whatsapp-play/commit/be0f76a160a696ceabd3ac3f9ce7c4ebc4c5e40f was commited, it was lacking changes made in https://github.com/rpotter12/whatsapp-play/pull/119. Anyway, I don't see why this could fail the CircleCI build.

> * Re-trigger the circle ci build for this PR? It has a weird error at the moment

I did it but it does not seem to fix the build.

I think we could merge this pull request if the tests are passing locally, and take action depending on whether future pull requests fail CircleCI build or not.

**Comment by nileshpatra** [comment]
@rpotter12 did you enable [this](https://circleci.com/docs/2.0/oss/#build-pull-requests-from-forked-repositories)? 
```
To enable CircleCI for forked pull requests, we go to the settings page
 for our project within CircleCI and 
choose Build Settings > Advanced Settings and 
enable the Build forked pull requests option.
```

**Comment by nileshpatra** [comment]
@rpotter12 I pulled and made a PR to my local fork, and circleci builds pass.
check [here](https://github.com/nileshpatra/whatsapp-play/pull/1)
This is likely, an error in the circleci option setup

**Comment by rpotter12** [comment]
> @rpotter12 did you enable [this](https://circleci.com/docs/2.0/oss/#build-pull-requests-from-forked-repositories)?
> 
> ```
> To enable CircleCI for forked pull requests, we go to the settings page
>  for our project within CircleCI and 
> choose Build Settings > Advanced Settings and 
> enable the Build forked pull requests option.
> ```

@nileshpatra Build forked pull requests options was already enabled.

> @rpotter12 I pulled and made a PR to my local fork, and circleci builds pass.
> check here
> This is likely, an error in the circleci option setup

I think we should merge this pull request as it passing locally.

**Comment by rpotter12** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 1
- Added 3
           

Complexity increasing per file
==============================
- wplay/utils/browser_config.py  4
- wplay/utils/target_data.py  2
- wplay/utils/kill_process.py  3
- wplay/utils/target_search.py  7
         

Complexity decreasing per file
==============================
+ wplay/onlinetracker.py  -1
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/rpotter12/whatsapp-play/pullRequest?prid=5051926&bid=16632931)

File: wplay/messagetimer.py (hunk 5 above)
Comment (rpotter12) [inline wplay/messagetimer.py]: Codacy Issue found: [Standard pseudo-random generators are not suitable for security/cryptographic purposes.](https://app.codacy.com/manual/rpotter12/whatsapp-play/pullRequest?prid=5051926)

File: wplay/utils/helpers.py [hunk 6]
```diff
@@ -10,18 +10,25 @@
...
```
Comment (rpotter12) [inline wplay/utils/helpers.py]: Codacy Issue found: [Trailing whitespace](https://app.codacy.com/manual/rpotter12/whatsapp-play/pullRequest?prid=5051926)

File: wplay/utils/target_data.py (hunk 4 above)
Comment (rpotter12) [inline wplay/utils/target_data.py]: Codacy Issue found: [No exception type(s) specified](https://app.codacy.com/manual/rpotter12/whatsapp-play/pullRequest?prid=5051926)

**Pull request merged**